src/libmongoc/tests/test-mongoc-scram.c

```diff
@@ -130,6 +145,35 @@ test_mongoc_scram_sasl_prep (void)
    }
 #endif
 }
+
+static void
+test_mongoc_utf8_char_length (void)
+{
+   ASSERT_CMPINT (_mongoc_utf8_char_length (","), ==, 1u);
+   ASSERT_CMPINT (_mongoc_utf8_char_length ("ɶ"), ==, 2u);
+   ASSERT_CMPINT (_mongoc_utf8_char_length ("ྡྷ"), ==, 3u);
+   ASSERT_CMPINT (_mongoc_utf8_char_length ("🌂"), ==, 4u);
+}
+
+static void
+test_mongoc_utf8_string_length (void)
+{
+   ASSERT_CMPINT (_mongoc_utf8_string_length (",ase"), ==, 4u);
```

Comment:
```suggestion
   ASSERT_CMPSIZE_T (_mongoc_utf8_string_length (",ase"), ==, 4u);
```
Use `ASSERT_CMPSIZE_T` to compare `size_t`.